File: Pascal/Prog_exe_01.pas
Program Prog_exe_01 ;

Var
	nota1, nota2, nota3, nota4:integer;

 Begin
 
 	Writeln('Programa para o cálculo de média anual de notas:');
 	Writeln('Escreva a nota do 1º bim:');
 	readln(nota1);
 	writeln('Escreva a nota do 2º bim:');
 	readln(nota2);
 	writeln('Escreva a nota do 3º bim:');
 	readln(nota3);
 	writeln('Escreva a nota do 4º bim:');
 	readln(nota4);
 	
 	writeln('A média anual deste aluno é: ',(nota1 + nota2 + nota3 + nota4)/4);
  
 End.

File: C/Prog_exe_01.c
#include <stdio.h>

int main(int argc, char const *argv[])
{

	int nota1, nota2, nota3, nota4;
	float media;

	printf("%s\n", "Programa para o cálculo de média anual de notas:");
	printf("%s\n", "Escreva a nota do 1º bim:");
	scanf("%d", &nota1);
 	printf("%s\n", "Escreva a nota do 2º bim:");
	scanf("%d", &nota2);
	printf("%s\n", "Escreva a nota do 3º bim:");
	scanf("%d", &nota3);
	printf("%s\n", "Escreva a nota do 4º bim:");
	scanf("%d", &nota4);

	media = (nota1 + nota2 + nota3 + nota4) / 4;

	printf("%s%f\n", "A média anual deste aluno é: ", media);

	return 0;
}